internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -798,12 +798,12 @@ func convertPublish(topics []manifest.Topic, accountID, region, app, env, svc st
 	// convert the topics to template Topics
 	for _, topic := range topics {
 		var fifoConfig *template.FIFOTopicConfig
-		if !topic.FIFO.IsEmpty() {
-			fifoConfig = &template.FIFOTopicConfig{
-				Enable: topic.FIFO.Enable,
-			}
+		if topic.FIFO.IsEnabled() {
+			fifoConfig = &template.FIFOTopicConfig{}
```

Comment:
nit: perhaps we could
```suggestion
		if topic.FIFO.IsEnabled() {
			fifoConfig = &template.FIFOTopicConfig{
			         ContentBasedDeduplication: topic.FIFO.Advanced.ContentBasedDeduplication,
			}
		}
```